neo4j-quickstart/pom.xml

```diff
@@ -15,7 +15,7 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <testcontainers.version>1.12.5</testcontainers.version>
+    <testcontainers.version>1.14.1</testcontainers.version>
```

Comment:
It's in the BOM so maybe we should just drop the version?

Reply:
Yeah, I saw the BOM had testcontainers after I made the change. I'll update

Reply:
Actually, it's not in the BOM, yet :wink: https://github.com/quarkusio/quarkus/pull/9073